tests/unit/Services/ReapplyServiceTest.php

```
@@ -0,0 +1,539 @@
+
+    public function testPostWriteHashFailureSkipsLockUpdate(): void
+    {
+        $this->seedTracked('config/params.php', "stub\n", "stub\n");
+
+        MockerState::addCondition(
+            'yii\\scaffold\\Scaffold\\Lock',
+            'hash_file',
+            [],
+            false,
+            default: true,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testPostWriteHashFailureSkipsLockUpdate'. (undefined)

(StaticAccess)